code/src/index.css

```diff
@@ -11,3 +15,130 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.MovieContainer {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+}
+
+.MovieWrapper {
+  position: relative;
+  width: 100%;
+  background-color: #000;
+}
+
+.MovieImage {
+  height: 600px;
+  width: 100%;
+  object-fit: cover;
+}
+
+.MovieOverlay {
+  color: #fff;
+  padding: 15px;
+  word-wrap: break-word;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 1));
+}
+
+.MovieTitle {
+  margin: 0;
+  font-size: 25px;
+}
+
+.releasedate {
+  text-transform: uppercase;
+}
+
+.home {
+  position: fixed;
```

Comment:
A suggestion is to remove the postion fixed if you don't want the button to move when you scroll on your page.